service_mesh/v2x/upgrading-ossm.adoc

```diff
@@ -0,0 +1,455 @@
+
+Security Discovery Service (SDS) is used to distribute certificates and keys to sidecars directly from istiod. In {ProductName} version 1.1, secrets were generated by Citadel, which were used by the proxies to retrieve their client certificates and keys.
+
+[id="ossm-upgrading-differences-annotation_{context}"]
+=== Annotation changes
+
+The following annotations are no longer supported in v2.0. If you are using one of these annotations, you must update your workload before moving it to a v2.0 control plane.
+
+* `sidecar.maistra.io/proxyCPULimit` has been replaced with `sidecar.istio.io/proxyCPULimit`.
+* `sidecar.maistra.io/proxyMemoryLimit` has been replaced with `sidecar.istio.io/proxyMemoryLimit`
+* `sidecar.istio.io/discoveryAddress` is no longer supported. Also, the default discovery address has moved from `pilot.<control_plane_namespace>.svc:15010` (or port 15011, if mtls is enabled) to `istiod-<smcp_name>.<control_plane_namespace>.svc:15012`.
+* The health status port is no longer configurable and is hard-coded to 15021.  * If you were defining a custom status port, for example, `status.sidecar.istio.io/port`, you must remove the override before moving the workload to a v2.0 control plane. Readiness checks can still be disabled by setting the status port to `0`.
+* Kubernetes Secret resources are no longer used to distribute client certificates for sidecars. Certificates are now distributed through istiod’s SDS service. If you were relying on mounted secrets, they are longer available for workloads in v2.0 control planes.
+
+[id="ossm-upgrading-differences-behavior_{context}"]
+=== Behavioral changes
+
+Some features in {ProductName} 2.0 work differently than they did in previous versions.
+
+* The readiness port on gateways has moved from `15020` to `15021`.
+* The target host visibility includes VirtualService, as well as ServiceEntry resources. It includes any restrictions applied through Sidecar resources.
+* Automatic mutual TLS is enabled by default. Proxy to proxy communication is automatically configured to use mTLS, regardless of global PeerAuthentication policies in place.
+* Secure connections are always used when proxies communicate with the control plane regardless of `spec.security.controlPlane.mtls` setting. The `spec.security.controlPlane.mtls` setting is only used when configuring connections for Mixer telemetry or policy.
+
+[id="ossm-upgrading-mig-unsupp_{context}"]
+=== Migration details for unsupported resources
+
+.Policy (authentication.istio.io/v1alpha1)
+
+Policy resources must be migrated to new resource types for use with v2.0 control planes.  Depending on the specific configuration in your Policy resource, you may have to configure multiple resources to achieve the same effect.
```

Comment:
Can we list those resources here, `PeerAuthentication` and/or `RequestAuthentication`